apis/core/test/domain/dataset/update.test.ts

```diff
@@ -5,12 +5,14 @@ import $rdf from 'rdf-ext'
 import { dcat, dcterms, rdf, schema, sh, _void } from '@tpluscode/rdf-ns-builders'
 import { TestResourceStore } from '../../support/TestResourceStore'
 import { update } from '../../../lib/domain/dataset/update'
+import { cube } from '@cube-creator/core/namespace'
 
 describe('domain/dataset/update', () => {
   let store: TestResourceStore
   const dataset = clownface({ dataset: $rdf.dataset(), term: $rdf.namedNode('dataset') })
     .addOut(dcterms.title, 'My Title')
     .addOut(dcat.keyword, 'Test')
+  const cubeNode = dataset.namedNode('cube').addOut(rdf.type, cube.Cube)
```

Comment:
Almost there. I would only just add one more triple to ensure that the cube and dataset remain connected
```suggestion
  const cubeNode = dataset.namedNode('cube')
    .addOut(rdf.type, cube.Cube)
    .addIn(schema.hasPart, dataset)
```